text/0000-parse-result-meta-data.md

```diff
@@ -0,0 +1,78 @@
+- Start Date: 2015-11-20
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This RFC outlines ideas in adding meta data into into elements through the
+means of meta links and adding link relations to the Parse Result namespace.
+
+# Motivation
+
+Parsing happens in multiple steps and information is added by different layers
+in the stack in some situations. It would be helpful for end users to have meta
+data to let the user know where data came from and which information was added
+that did not exist in the original representation.
+
+The idea with proposing meta links would be to provide a generic way to link to
+out-of-band information to allow the user to explore more information in a
```

Comment:
Not out of band if you can link to it. Maybe "link to what would normally be out-of-band ..."